pkgs/by-name/co/conduktor-ctl/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "conduktor-ctl";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "conduktor";
+    repo = "ctl";
+    rev = "v${version}";
+    hash = "sha256-1keo9nL+vC8jYSvKkEyj2ridhKyu7ekyTJTk0/7gTHY=";
+  };
+
+  vendorHash = "sha256-HoIRw72Rxonwozqqz8z+YtZDKIJ6k5pqjf/EQFkhDik=";
+
+  # ldflags = [ "-o" "conduktor" ];
+
+  doCheck = false;
```

Comment:
Ideally this should have a comment explaining why

Reply:
Thank you for noticing, this was actually forgot after some testing and has now been removed as not needed